src/Fhir.Anonymizer.Tool/configuration-sample.json

```diff
@@ -1,552 +1,192 @@
+    {"path": "Contract.term.text", "method": "redact"},
+    {"path": "Contract.term.offer.answer.value", "method": "redact"},
+    {"path": "Contract.term.offer.text", "method": "redact"},
+    {"path": "Contract.term.offer.linkId", "method": "redact"},
+    {"path": "Contract.term.asset.context.text", "method": "redact"},
+    {"path": "Contract.term.asset.condition", "method": "redact"},
+    {"path": "Contract.term.asset.text", "method": "redact"},
+    {"path": "Contract.term.asset.linkId", "method": "redact"},
+    {"path": "Contract.term.asset.answer", "method": "redact"},
+    {"path": "Contract.term.asset.valuedItem.entity", "method": "redact"},
+    {"path": "Contract.term.asset.valuedItem.payment", "method": "redact"},
+    {"path": "Contract.term.asset.valuedItem.linkId", "method": "redact"},
+    {"path": "Contract.term.action.linkId", "method": "redact"},
+    {"path": "Contract.term.action.contextLinkId", "method": "redact"},
+    {"path": "Contract.term.action.requesterLinkId", "method": "redact"},
+    {"path": "Contract.term.action.performerLinkId", "method": "redact"},
+    {"path": "Contract.term.action.reason", "method": "redact"},
+    {"path": "Contract.term.action.reasonLinkId", "method": "redact"},
+    {"path": "Contract.term.group", "method": "redact"},
+    {"path": "ExplanationOfBenefit.disposition", "method": "redact"},
+    {"path": "ExplanationOfBenefit.preAuthRef", "method": "redact"},
+    {"path": "ExplanationOfBenefit.supportingInfo.value", "method": "redact"},
+    {"path": "ExplanationOfBenefit.insurance.preAuthRef", "method": "redact"},
+    {"path": "ExplanationOfBenefit.accident.location", "method": "redact"},
+    {"path": "ExplanationOfBenefit.item.location", "method": "redact"},
+    {"path": "ExplanationOfBenefit.addItem.location", "method": "redact"},
+    {"path": "ExplanationOfBenefit.processNote.text", "method": "redact"},
+    {"path": "ExplanationOfBenefit.benefitBalance.name", "method": "redact"},
+    {"path": "ExplanationOfBenefit.benefitBalance.description", "method": "redact"},
+    {"path": "ExplanationOfBenefit.benefitBalance.financial.allowed", "method": "redact"},
+    {"path": "InsurancePlan.name", "method": "redact"},
+    {"path": "InsurancePlan.alias", "method": "redact"},
+    {"path": "InsurancePlan.coverage.benefit.requirement", "method": "redact"},
+    {"path": "InsurancePlan.plan.generalCost.comment", "method": "redact"},
+
+    {"path": "nodesByType('Attachment').title", "method": "redact"},
+    {"path": "nodesByType('Attachment').url", "method": "redact"},
+    {"path": "nodesByType('CodeableConcept').text", "method": "redact"},
+    {"path": "nodesByType('Coding').display", "method": "redact"},
+    {"path": "nodesByType('Coding').code", "method": "redact"},
+    {"path": "nodesByType('ContactDetail').name", "method": "redact"},
+    {"path": "nodesByType('Identifier').value", "method": "redact"},
+    {"path": "nodesByType('Narrative').div", "method": "redact"},
```

Comment:
Shall we redact whole Narrative to keep consistency with previous version and avoid invalid output here?

Reply:
as we offline discussed, we should keep same behavior as before.